templates-blocks/cards/card.php

```diff
@@ -81,7 +85,13 @@
 		<img class="card-img-top" src="<?php echo $image_url; ?>" alt="<?php echo $image_alt; ?>">
 	<?php endif; ?>
 	<?php if ('icon' == $header_style) : ?>
-		<i class="fas fa-<?php echo $icon_name; ?> fa-2x card-icon-top"></i>
+		<?php if ('maroon' == $icon_color || 'gold' == $icon_color) : ?>
+			<span class="text-<?php echo $icon_color; ?>">
+		<?php endif; ?>
+		<span class="fas fa-<?php echo $icon_name; ?> fa-2x card-icon-top"></span>
```

Comment:
The color works with no issues. 
I just want to suggest that we change the $icon_name here to take the whole thing, like "fas fa-user" or fab fa-facebook" and so..

Reply:
Yes, agreed. There are some free icons prepended with `fab` I have run across in v4 free version used in webspark if I remember correctly, not sure if that has changed in v5 or higher.
I agree to take the whole name for compatibility with all icons, just in case.

Reply:
I updated the code to require the full icon name per your notes. I was worried that legacy icon cards might fail, so I put some checks in there to do the following:

- Look to see if the string starts with `fas` or `fab` as we now expect it to
- If it **does not** start with `fas` or `fab`, presume it's an old icon card with only the base icon name
- Clean it up and prepend `fas fa-` like the old code used to

Here's a screenshot showing three cards working, each with a different format for the string:
- `fab fa-apple` (the new way)
- `fa-toilet` (neither way, but possible)
- `user` (the old way)

![Screen Shot 2022-12-08 at 5 57 39 PM](https://user-images.githubusercontent.com/31013359/206599598-3497d6ad-b1ab-4c0d-b2d9-b6102ef6970b.png)